canonical-web-and-design/dotrun
# Add multipass script

## Done

Added multipass script with support for Mac and Linux, it will:
- Install multipass if missing
- Create a dotrun instance in multipass
- Create and share a `dotrun-projects` folder between machines
- Install a dotrun "alias" for bash or zsh

Fixes #37 

## QA

Please QA in a Mac computer without multipass installed

```bash
curl -s https://github.com/canonical-web-and-design/dotrun/blob/master/scripts/install-with-multipass.sh | bash
```

Check it finish correctly and them clone one of our web projects to the new folder `dotrun-projects`.
```
cd ~/dotrun-projects
git clone <SOME PROJECT BUT NOT UBUNTU.COM>
cd <PROJECT>
dotrun
```
